staging/src/k8s.io/mount-utils/mount_helper_unix.go

```diff
@@ -32,7 +32,7 @@ const (
 	// At least number of fields per line in /proc/<pid>/mountinfo.
 	expectedAtLeastNumFieldsPerMountInfo = 10
 	// How many times to retry for a consistent read of /proc/mounts.
-	maxListTries = 3
+	maxListTries = 10
```

Comment:
what are the performance implications of tripling this?

Reply:
Almost none, I was able to get tens of thousands of /proc/mounts reads per second. So... under 1 ms?